src/error_handling.c

```diff
@@ -40,6 +40,9 @@ RCUTILS_THREAD_LOCAL rcutils_error_state_t gtls_rcutils_error_state;
 RCUTILS_THREAD_LOCAL bool gtls_rcutils_error_string_is_formatted = false;
 RCUTILS_THREAD_LOCAL rcutils_error_string_t gtls_rcutils_error_string;
 RCUTILS_THREAD_LOCAL bool gtls_rcutils_error_is_set = false;
+RCUTILS_THREAD_LOCAL bool gtls_rcutils_error_code_is_formatted = false;
+RCUTILS_THREAD_LOCAL rcutils_ret_t gtls_rcutils_error_code = 21;
+RCUTILS_THREAD_LOCAL bool gtls_rcutils_error_code_is_set = true;
```

Comment:
is this intention to initialize into `true`?

```suggestion
RCUTILS_THREAD_LOCAL bool gtls_rcutils_error_code_is_set = false;
```

Reply:
Please see the updated code.